src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml

```diff
@@ -69,5 +69,73 @@
                 <attr name="position" intvalue="900"/>
             </file>
         </folder>
+        
```

Comment:
Are you really sure that the toolbar is not visible, when I do not have a remote project?